app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -75,6 +79,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return v;
     }
 
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+    }
+
```

Comment:
We dont need this method that only calls super.

Reply:
Removing it (Y)